src/interpolate/piecewise.d.ts

```diff
@@ -0,0 +1,5 @@
+declare function interpolatorPiecewise(
```

Comment:
I'm not 100% sure that this type declaration is 100% corresponds to the `interpolatorPiecewise` function.